Lib/ast.py

```diff
@@ -556,14 +556,37 @@ def __new__(cls, *args, **kwargs):
 # We unparse those infinities to INFSTR.
 _INFSTR = "1e" + repr(sys.float_info.max_10_exp + 1)
 
+
 class _Unparser(NodeVisitor):
     """Methods in this class recursively traverse an AST and
     output source code for the abstract syntax; original formatting
     is disregarded."""
 
+    PRECEDENCE_LEVELS = {
```

Comment:
You may use an enum for this.

Reply:
Importing `enum` will make importing the module slower, and some people were concerned about that.

Reply:
Is there any chance we can split `ast` with `unparse` like `collections` and `collections.abc`? I believe in that way code will be much more readable and easy to maintain.

Reply:
I don't think having a module for only one function is worth enough. At that point the trade-off of the imports becomes similar

Reply:
Well, that's why I proposed to put the unparse feature in a separated ast.unparse submodule in the first place ;-)

Reply:
> Well, that's why I proposed to put the unparse feature in a separated ast.unparse submodule in the first place ;-)

I still think is not a good idea to have a separate module for just one function being the reason just that we want to import things.

Reply:
@pablogsal if `ast.unparse import unparse` feels weird, what about lazy loading. We can create a module named `_ast_unparse` etc. and then load `unparse` function if needed by module level getattr.